packages/stark-ui/src/modules/toast-notification/services/toast-notification.service.ts

```diff
@@ -0,0 +1,116 @@
+import { ApplicationRef, Inject, Injectable, TemplateRef } from "@angular/core";
+import { MatSnackBar, MatSnackBarConfig, MatSnackBarDismiss, MatSnackBarRef, MatSnackBarVerticalPosition, MatSnackBarHorizontalPosition } from "@angular/material";
+import { Observable, Observer } from "rxjs";
+import { map } from "rxjs/operators";
+import { STARK_LOGGING_SERVICE, StarkLoggingService } from "@nationalbankbelgium/stark-core";
+import {
+	starkToastNotificationServiceName,
+	StarkToastNotificationService,
+	StarkToastNotificationOptions,
+	StarkToastNotificationResult
+} from "../services";
+import {
+	StarkToastNotificationComponent,
+	StarkToastNotificationMessage
+} from "../components";
+
+/**
+ * Service to display different types of messages in a toast (info, warning or error).
+ */
+@Injectable()
+export class StarkToastNotificationServiceImpl implements StarkToastNotificationService {
+	public config: MatSnackBarConfig;
+	public template: TemplateRef<any>;
+	public obs: Observer<StarkToastNotificationResult>;
+	public toast: MatSnackBarRef<StarkToastNotificationComponent>;
+	public _defaultToastOptions: StarkToastNotificationOptions = {
+		delay: 3000,
```

Comment:
These default options should be the same as in the Stark old: 5000 delay and "top right" position.  This is to reduce the amount of changes the developers will have to do when migrating their app

Also, these defaults should be customizable, for that you can add a config object as parameter in the `forRoot()` of the `StarkToastNotificationModule` and pass it to the constructor of the `StarkToastNotificationServiceImpl`